axelar-chains-config/info/stagenet.json

```diff
@@ -1704,7 +1822,8 @@
         },
         "hedera": {
           "executeProposalId": "48"
-        }
+        },
+        "sui": {}
```

Comment:
after some discussion with Milap we decided to remove the saving of the `executeProposalId`, we thought the additional information was not that useful and it could be retrieved from axelarscan anyway

Reply:
yes, creation of empty objects with key `chainName` fixed with https://github.com/axelarnetwork/axelar-contract-deployments/pull/472, those objects are created when passing the option `-n` when it shouldn't